--- ./debian/rules
+++ ./debian/rules
@@ -17,8 +17,6 @@
 
 clean:
 	@ echo CLEAN
-	rm -f ../$(upstreampck)
-	tar czf ../$(upstreampck) --exclude=debian --exclude=.pc .
 	dh_testdir
 	dh_auto_clean
 	dh_clean
@@ -29,19 +27,21 @@
 		-DCMAKE_BUILD_TYPE=RELEASE \
 		-DCMAKE_INSTALL_PREFIX=/usr \
 		-DENABLE_BMV2=ON \
-		-DENABLE_EBPF=ON \
-		-DENABLE_UBPF=ON \
-		-DENABLE_DPDK=ON \
-		-DENABLE_P4C_GRAPHS=ON \
-		-DENABLE_P4TEST=ON \
+		-DENABLE_EBPF=OFF \
+		-DENABLE_UBPF=OFF \
+		-DENABLE_DPDK=OFF \
+		-DENABLE_P4C_GRAPHS=OFF \
+		-DENABLE_P4TEST=OFF \
 		-DENABLE_DOCS=OFF \
 		-DENABLE_GC=ON \
 		-DENABLE_GTESTS=OFF \
 		-DENABLE_PROTOBUF_STATIC=ON \
 		-DENABLE_MULTITHREAD=OFF \
-		-DENABLE_TEST_TOOLS=ON \
+		-DENABLE_TEST_TOOLS=OFF \
 		-DENABLE_GMP=ON
 
+override_dh_auto_build:
+	dh_auto_build --builddirectory=build/ -- -j1
 
 override_dh_auto_install:
-	dh_auto_install --builddirectory=build/ -- DESTDIR=$(CURDIR)/debian/p4lang-p4c install
+	dh_auto_install --builddirectory=build/ -- -j1 DESTDIR=$(CURDIR)/debian/p4lang-p4c install
