--- a/debian/rules
+++ b/debian/rules
@@ -18,7 +18,5 @@
 clean:
 	@ echo CLEAN
-	rm -f ../$(upstreampck)
-	tar czf ../$(upstreampck) --exclude=debian --exclude=.pc .
 	dh_testdir
 	dh_auto_clean
 	dh_clean
@@ -29,7 +27,7 @@
 override_dh_auto_build:
 	# Explicit 'optional' labels are disallowed in the Proto3 syntax.
 	# https://github.com/protocolbuffers/protobuf/blob/main/docs/field_presence.md
 	sed -i 's/optional //g' proto/p4runtime/proto/p4/config/v1/p4info.proto
-	$(MAKE) all
+	env MAKEFLAGS= $(MAKE) -j1 all
 
 override_dh_auto_install:
 	dh_auto_install -- DESTDIR=$(CURDIR)/debian/p4lang-pi install
